app/models/invoice.rb

```diff
@@ -2,13 +2,15 @@
 require 'result_not_sold'
 
 class Invoice < ApplicationRecord
+  include BillingHelpers
```

Comment:
I would avoid `Helper` word. It doesn't explain the role an object plays. Consider refactoring. Maybe extract `address` and `vat_rate` into their own objects. As a quick fix they might remain right in `Invoice`. Otherwise by looking at `Invoice` class I have no idea what that `BillingHelpers` includes.